README.md

```diff
@@ -79,11 +79,22 @@ This project Iceberg also has modules for adding Iceberg support to processing e
 
 ### Compatibility
 
-Iceberg's Spark integration is compatible with Spark 2.4 and Spark 3.0 using the modules in the following table:
-
-| Iceberg version | Spark 2.4.x   | Spark 3.0.x    |
-| --------------- | ------------- | -------------- |
-| master branch   | spark-runtime | spark3-runtime |
-| 0.9.0           | spark-runtime | spark3-runtime |
-| 0.8.0           | spark-runtime |                |
+Iceberg's Spark integration is compatible with Spark using the modules in the following table:
+
+| Iceberg version | Spark 2.4.x           | Spark 3.0.x            | Spark 3.1.x                    | Spark 3.2.x                    |
+| --------------- | --------------------- | ---------------------- | ------------------------------ | ------------------------------ |
+| master branch   | iceberg-spark-runtime | iceberg-spark3-runtime | iceberg-spark-runtime-3.1_2.12 | iceberg-spark-runtime-3.2_2.12 |
+| 0.12.x          | iceberg-spark-runtime | iceberg-spark3-runtime | iceberg-spark-runtime-3.1_2.12 |                                |
+| 0.11.x          | iceberg-spark-runtime | iceberg-spark3-runtime |                                |                                |
+| 0.10.x          | iceberg-spark-runtime | iceberg-spark3-runtime |                                |                                |
+| 0.9.x           | iceberg-spark-runtime | iceberg-spark3-runtime |                                |                                |
+
+Iceberg's Flink integration is compatible with Flink using the modules in the following table:
+
+| Iceberg version | Flink 1.11.x          | Flink 1.12.x          | Flink 1.13.x          | Flink 1.14.x          |
+| --------------- | --------------------- | --------------------- | --------------------- | --------------------- |
+| master branch   | iceberg-flink-runtime | iceberg-flink-runtime | iceberg-flink-runtime | iceberg-flink-runtime |
```

Comment:
I think there were similar renames for Flink Jars. Can you take a look and update this table to point to the correct artifacts?

Reply:
done.please take a look, thx.